rules.ini

```
@@ -121,6 +121,7 @@ Snowdrop = \.sdfdata$
 Solar2D = (?:^|/)CoronaLabs\.Corona\.Native\.dll$
 Source = (?:^|/)(?:vphysics|bsppack)\.(?:dylib|dll|so)$
 Source2 = (?:^|/)gameinfo\.gi$
+Stencyl = assets/data/(?:behaviors|game|resources|scenes)\.mbs$
```

can be reduced to this
```suggestion
Stencyl = (?:^|/)assets/data/behaviors\.mbs$
```